.travis.yml

```diff
@@ -1,5 +1,24 @@
+services:
+    - docker
+before_script:
+    - docker login -u $DOCKER_USER -p $DOCKER_PASSWORD
+    - curl https://cli-assets.heroku.com/install.sh | sh  #install heroku
+    - docker login --username=$HEROKU_USER --password=$HEROKU_API_KEY registry.heroku.com
 script:
     - docker build --target test --tag my-test-image .
     - docker run my-test-image tests/unit
     - docker run my-test-image tests/integration
-    - docker run --env apiKey --env apiToken --env boardId --env TODO_LIST_ID --env DOING_LIST_ID --env DONE_LIST_ID --env FLASK_APP --env FLASK_ENV my-test-image tests/endtoend
\ No newline at end of file
+    - docker run --env apiKey --env apiToken --env boardId --env TODO_LIST_ID --env DOING_LIST_ID --env DONE_LIST_ID --env FLASK_ENV my-test-image tests/endtoend
+    - docker build --target production --tag $DOCKER_USER/todo-app:latest .  
+    - docker push $DOCKER_USER/todo-app:latest
+    # Get the latest image from Docker Hub (built by your CI pipeline) $ docker pull <user_name>/<image_name>
+    # Tag it for Heroku
+    - docker tag $DOCKER_USER/todo-app:latest registry.heroku.com/$HEROKU_APP/web
+    # Push it to Heroku registry
+    - docker push registry.heroku.com/$HEROKU_APP/web
+deploy:   
+    provider: script
+    script:    
+        - heroku container:release web --app $HEROKU_APP
```

Comment:
This looks good and travis seems happy, but we should add a link to the readme (particularly because all of your variables are secret) - I'm not able to verify that the app is on heroku & working as I don't know the address

Reply:
Added link in Readme